app/services/import_user.rb

```diff
@@ -1,4 +1,4 @@
-class PatientImport::ImportUser
+class ImportUser
```

Comment:
Why has this service moved?

Reply:
This import module was created and used for Patient import via CSV. I think the scope of this import user has now expanded beyond being a utility for only importing patients—it's now also used by all the FHIR importers for all kinds of resources. So I thought of moving this out.

It also fixes an awkward dependency graph where FHIR import services depend on an internal module of the `PatientImport` service.